telemetry_examples/src/main/java/com/newrelic/telemetry/examples/TelemetryClientExample.java

```diff
@@ -35,12 +35,19 @@
 public class TelemetryClientExample {
 
   public static void main(String[] args) throws Exception {
-    String insightsInsertKey = args[0];
+    // String insightsInsertKey = args[0];
+    String licenseKey = args[0];
 
-    // create a TelemetryClient with an http connect timeout of 10 seconds.
+    /*
     TelemetryClient telemetryClient =
         TelemetryClient.create(
             () -> new OkHttpPoster(Duration.of(10, ChronoUnit.SECONDS)), insightsInsertKey);
+    */
+
+    // create a TelemetryClient with an http connect timeout of 10 seconds.
+    TelemetryClient telemetryClient =
+        TelemetryClient.create(
+            () -> new OkHttpPoster(Duration.of(10, ChronoUnit.SECONDS)), licenseKey);
```

Comment:
I tried this with my prod license key and I'm getting 403. I don't think the code path will allow a license key in this particular example.  Did you get this to work with a license key?

Reply:
@yamnihcg  I don't think a license key will work here, without some changes to the create of the TelemetryClient.  

Lets update the TelemetryClient so that it configured with a license key